### PR TITLE
[plugin.video.drnu] 5.0.12

### PR DESCRIPTION
**Pull request by msj33**
### Description
- Update of plugin.video.drnu to version 5.0.12
- Change of addon owner from twinther -> msj33/community based(Details here: https://github.com/xbmc-danish-addons/plugin.video.drnu/issues/20#issuecomment-460169727 )
- Replaced strings.xml with strings.po for supporting the new language format.
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [O] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0 - Only 1 commit is the change - the other 2 are simple " typos" :-)

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.drnu/addon.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#
-#      Copyright (C) 2014 Tommy Winther
-#      http://tommy.winther.nu
```
Comment (Rechi) [inline plugin.video.drnu/addon.py]: The old coprytight infor should be kept. You can add yourself like you already did.

File: plugin.video.drnu/addon.xml [hunk 2]
```diff
@@ -11,16 +11,16 @@
...
         <disclaimer lang="en">Some parts of this addon may not be legal in your country of residence - please check with your local laws.</disclaimer>
         <disclaimer lang="da">Nogle dele af denne addon er muligvis ikke lovlig i dit land - kontroller venligst dine lokale love.</disclaimer>
         <license>GPL 2.0</license>
         <platform>all</platform>
         <language>da</language>
-        <email>tommy@winther.nu</email>
-        <website>http://tommy.winther.nu</website>
-        <forum>http://tommy.winther.nu/wordpress/forums/forum/addons/dr-nu-player/</forum>
+        <email>msj33@hotmail.com</email>
+        <website>https://github.com/xbmc-danish-addons/plugin.video.drnu</website>
```
Comment (Rechi) [inline plugin.video.drnu/addon.xml]: Website should list the site where the content is comming from.
So in this case something like dr.dk.

File: plugin.video.drnu/clearfavorites.py [hunk 3]
```diff
@@ -1,6 +1,5 @@
-#
-#      Copyright (C) 2014 Tommy Winther
-#      http://tommy.winther.nu
+#  Copyright (C) 2014 msj33
+#  https://github.com/xbmc-danish-addons/plugin.video.drnu
```
Comment (Rechi) [inline plugin.video.drnu/clearfavorites.py]: same

File: plugin.video.drnu/resources/language/resource.language.da_dk/strings.po [hunk 4]
```diff
@@ -0,0 +1,157 @@
+# Kodi Media Center language file
```
Comment (Rechi) [inline plugin.video.drnu/resources/language/resource.language.da_dk/strings.po]: Gotham isn't compatible with the new language directory layout (resource.language.*).
So this file still has to be in the same folder as the strings.xml file was (`plugin.video.drnu/resources/language/Danish/strings.so`).

File: plugin.video.drnu/resources/language/resource.language.en_gb/strings.po [hunk 5]
```diff
@@ -0,0 +1,157 @@
+# Kodi Media Center language file
```
Comment (Rechi) [inline plugin.video.drnu/resources/language/resource.language.en_gb/strings.po]: Gotham isn't compatible with the new language directory layout (resource.language.*).
So this file still has to be in the same folder as the strings.xml file was (`plugin.video.drnu/resources/language/English/strings.so`).

File: plugin.video.drnu/tvapi.py [hunk 6]
```diff
@@ -1,6 +1,5 @@
-#
-#      Copyright (C) 2014 Tommy Winther
-#      http://tommy.winther.nu
+#  Copyright (C) 2014 msj33
+#  https://github.com/xbmc-danish-addons/plugin.video.drnu
```
Comment (Rechi) [inline plugin.video.drnu/tvapi.py]: same

File: plugin.video.drnu/tvgui.py [hunk 7]
```diff
@@ -1,6 +1,5 @@
-#
-#      Copyright (C) 2014 Tommy Winther
-#      http://tommy.winther.nu
+#  Copyright (C) 2014 msj33
+#  https://github.com/xbmc-danish-addons/plugin.video.drnu
```
Comment (Rechi) [inline plugin.video.drnu/tvgui.py]: same

File: plugin.video.drnu/addon.xml [hunk 8]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.drnu" version="5.0.10" name="DR TV" provider-name="twinther">
+<addon id="plugin.video.drnu" version="5.0.12" name="DR TV" provider-name="msj33">
     <requires>
         <import addon="xbmc.python" version="2.1.0"/>
```
Comment (Rechi) [inline plugin.video.drnu/addon.xml]: > For gotham it is advised to set xbmc.python version to 2.14.0
```suggestion
         <import addon="xbmc.python" version="2.14.0"/>
```

**Comment by Rechi** [comment]
Please apply the following changes. Even better would be if you replace `print(...)` with `xbmc.log(...)`.
```
--- a/plugin.video.drnu/tvapi.py
+++ b/plugin.video.drnu/tvapi.py
@@ -135,7 +135,7 @@
                 url = url + '?' + urllib.urlencode(params, doseq=True)
 
             try:
-                print url
+                print(url)
             except:
                 pass
 
@@ -436,4 +436,4 @@
 
 if __name__ == '__main__':
     api = Api()
-    print api.programCardRelations('so-ein-ding', limit=50)
+    print(api.programCardRelations('so-ein-ding', limit=50))
```

File: plugin.video.drnu/resources/language/resource.language.da_dk/strings.po (hunk 4 above)
Comment (msj33) [reply plugin.video.drnu/resources/language/resource.language.da_dk/strings.po]: Should the filetype be .so? (typo?) or just be the exact same file and filetype.

File: plugin.video.drnu/addon.py (hunk 1 above)
Comment (msj33) [reply plugin.video.drnu/addon.py]: Re-added - But removed his private website

File: plugin.video.drnu/addon.xml (hunk 2 above)
Comment (msj33) [reply plugin.video.drnu/addon.xml]: Fixed

File: plugin.video.drnu/clearfavorites.py (hunk 3 above)
Comment (msj33) [reply plugin.video.drnu/clearfavorites.py]: Re-added - But removed his private website

File: plugin.video.drnu/resources/language/resource.language.da_dk/strings.po (hunk 4 above)
Comment (msj33) [reply plugin.video.drnu/resources/language/resource.language.da_dk/strings.po]: Copied new strings.po files back to original folder structure

File: plugin.video.drnu/resources/language/resource.language.en_gb/strings.po (hunk 5 above)
Comment (msj33) [reply plugin.video.drnu/resources/language/resource.language.en_gb/strings.po]: Copied new strings.po files back to original folder structure

File: plugin.video.drnu/tvapi.py (hunk 6 above)
Comment (msj33) [reply plugin.video.drnu/tvapi.py]: Re-added Tommy - Removed his personal website.

File: plugin.video.drnu/tvgui.py (hunk 7 above)
Comment (msj33) [reply plugin.video.drnu/tvgui.py]: Re-added Tommy - Removed his personal website.

File: plugin.video.drnu/addon.xml (hunk 8 above)
Comment (msj33) [reply plugin.video.drnu/addon.xml]: Bumped the Python version as recommended

**Comment by msj33** [comment]
Thanks for doing code review - Have changed codebase according to the recommendations.

File: plugin.video.drnu/resources/language/resource.language.da_dk/strings.po (hunk 4 above)
Comment (Rechi) [reply plugin.video.drnu/resources/language/resource.language.da_dk/strings.po]: `.po` is the correct excetnsion, `.so` is a mistake.
This file should be removed, only `plugin.video.drnu/resources/language/Danish/strings.po` should exist.

File: plugin.video.drnu/resources/language/resource.language.en_gb/strings.po (hunk 5 above)
Comment (Rechi) [reply plugin.video.drnu/resources/language/resource.language.en_gb/strings.po]: This file should be removed, only `plugin.video.drnu/resources/language/English/strings.po` should exist.

File: plugin.video.drnu/addon.xml (hunk 2 above)
Comment (Rechi) [reply plugin.video.drnu/addon.xml]: No, you now mixed source and website.
source is source code repo and was correct before
website is content website

File: plugin.video.drnu/resources/language/resource.language.da_dk/strings.po (hunk 4 above)
Comment (msj33) [reply plugin.video.drnu/resources/language/resource.language.da_dk/strings.po]: OK - Removed it in new commit

File: plugin.video.drnu/resources/language/resource.language.en_gb/strings.po (hunk 5 above)
Comment (msj33) [reply plugin.video.drnu/resources/language/resource.language.en_gb/strings.po]: OK - Removed it in new commit

File: plugin.video.drnu/addon.xml (hunk 2 above)
Comment (msj33) [reply plugin.video.drnu/addon.xml]: Your completely right :D - Fixed

**Comment by msj33** [comment]
Done the requested changes - Thanks

**Comment by Rechi** [comment]
Everting looks good now.
> Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Please squash your commits. If you need instructions on how to do that, fell free to ask.

**Comment by msj33** [comment]
Hope I did squash this right :-)

**Comment by msj33** [comment]
@Rechi - Thanks for all your help.

How long time does it normally take to be generally available to Kodi clients?

**Comment by Rechi** [comment]
Till it is in the repo at max 2 hours and 30 minutes.
Kodi clients check the repo for updates every 24 hours, but a check can be forced.

**Pull request merged**